core/broadleaf-framework/src/main/java/org/broadleafcommerce/core/catalog/domain/ProductImpl.java

```diff
@@ -741,21 +742,30 @@ public void setCategory(Category category) {
 
     @Override
     public Map<String, Media> getMedia() {
-        return getDefaultSku().getSkuMedia();
+        return getDefaultSku().getSkuMediaXref()
```

Comment:
I think it would be good to remove sku.getSkuMedia / setSkuMedia methods from Sku.
Also, from the product we can remove deprecated defaultParentCategoryId column...